modules/social_features/social_search/config/install/views.view.search_content.yml

```diff
@@ -280,16 +280,17 @@ display:
                   min: ''
                   max: ''
           plugin_id: search_api_date
-        search_api_language:
-          id: search_api_language
+        language_with_fallback:
+          id: language_with_fallback
           table: search_api_index_social_content
-          field: search_api_language
+          field: language_with_fallback
           relationship: none
           group_type: group
           admin_label: ''
+          plugin_id: search_api_language
           operator: in
           value:
-            '***LANGUAGE_language_interface***': '***LANGUAGE_language_interface***'
+            '***LANGUAGE_site_default***': '***LANGUAGE_site_default***'
```

Comment:
@volodymyr-sydor Do we really need to change value here? 

I see we have **LANGUAGE_language_interface**  for similar case [here](https://github.com/goalgorilla/cablecar/pull/537/files)

Reply:
@ribel Yes, we should change it, otherwise search by keyword won't work